globals/strings.gd

```diff
@@ -209,11 +210,21 @@ const SkillNameMap = {
 	"noelbeyondthedoor": "Noel ~Beyond the door~",
 	"marinebeyondthedoor": "Marine ~Beyond the door~",
 	"elementaryisntit": "Elementary, isn't it?",
-
 	"flowerrhapsody": "flower rhapsody",
 
-
-
+	"konnakiri": "Konnakiri~",
```

Comment:
Going forward, I wonder if we want to just use the skill tags as the key in the translations sheet, then we won't need this at all, right?

Reply:
Yeap. They feel redundant.